bukkit/src/main/java/me/lucko/luckperms/bukkit/BukkitCommandExecutor.java

```diff
@@ -79,6 +79,7 @@ public void register() {
 
     @Override
     public boolean onCommand(@NonNull CommandSender sender, @NonNull Command command, @NonNull String label, @NonNull String[] args) {
+
```

Comment:
Please can you remove the added newlines - there's one here and a couple of others in various places.

```suggestion
```